### ISSUE TITLE
History of vaginal douching

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9981]
**Preferred term label:**
History of vaginal douching

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bacterial vaginosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001093
- label: Exposure to vaginal douching
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: PMID:12762087
- def: A history of intravaginal cleansing with a liquid solution for personal hygiene or aesthetic reasons.
- comment: Douching represents a risk factor for bacterial vaginosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9981
- synonyms:
